src/js/background.js

```diff
@@ -144,6 +146,24 @@ function Badger() {
     return available;
   }
 
+  // checks for availability of alternateErrorPagesEnabled option on browser
+  function checkAlternateErrorPagesSupport() {
```

Comment:
I don't think we need these two functions ... We could do these checks where they are needed instead. This would avoid a bunch of unnecessary code. Instead of defining, looking up, message passing ... do the check directly on the options page.